src/module.api/poolpair.service.ts

```diff
@@ -60,7 +60,7 @@ export class PoolPairService {
       if (Object.values(result).length > 0) {
         return Object.values(result)[0]
       }
-    } catch (err) {
+    } catch (err: any) {
       if (err?.payload?.message !== 'Pool not found') {
```

Comment:
```suggestion
    } catch (err) {
      if (err instanceof RpcApiError && err?.payload?.message !== 'Pool not found') {
```
We can potentially use instanceof here to avoid casting errors to any

Reply:
Note that this requires `RpcApiError` to be imported so will need manual commits I think 🤔

Reply:
Quite reluctant to make changes here, since it is a generic issue and all over the codebase. I think we should open an issue to clean it up if we have to.

Reply:
I am wondering if we want to set the typescript rules to use the [useUnknownInCatchVariables](https://www.typescriptlang.org/tsconfig#useUnknownInCatchVariables) until we formally address this so we don't have to cast back to any for our try/catch.

But that could also be part of a separate PR.

Reply:
Omg, I didn't actually look deep into this. JS 😅 
![image](https://user-images.githubusercontent.com/44192990/161017214-dff4194c-8628-4fe4-869f-5b6ceb68b545.png)